include/interface/blas3_interface_sycl_gemm.hpp

```diff
@@ -0,0 +1,166 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) 2016 Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename blas3_interface_sycl.hpp
+ *
+ **************************************************************************/
+
+
+#ifndef BLAS3_INTERFACE_SYCL_GEMM_HPP
+#define BLAS3_INTERFACE_SYCL_GEMM_HPP
+
+
+#include <algorithm>
+#include <cctype>
+#include <cmath>
+#include <iostream>
+#include <stdexcept>
+#include <vector>
+
+
+// #define VERBOSE 1
```

Comment:
commented out code.. can we remove that?

Reply:
Removed.